cozy/syntax.py

```diff
@@ -85,12 +87,14 @@ def __repr__(self):
 ENative             = declare_case(Exp, "ENative",            ["e"])
 EEnumEntry          = declare_case(Exp, "EEnumEntry",         ["name"])
 ENull               = declare_case(Exp, "ENull")
+## TODO: I would name the fields `then_expr` and `else_expr` rather than `*_branch`.
 ECond               = declare_case(Exp, "ECond",              ["cond", "then_branch", "else_branch"])
 EBinOp              = declare_case(Exp, "EBinOp",             ["e1", "op", "e2"])
 EUnaryOp            = declare_case(Exp, "EUnaryOp",           ["op", "e"])
 EArgMin             = declare_case(Exp, "EArgMin",            ["e", "f"])
 EArgMax             = declare_case(Exp, "EArgMax",            ["e", "f"])
 EHandle             = declare_case(Exp, "EHandle",            ["addr", "value"])
+## TODO: I object to the use of the same name "f" here for "field" and in 3 other places for "function".  Change one or both.
```

Comment:
I am going to roll this and similar ideas into #45, since making this change is nontrivial.